CarpetLib/src/restrict_3d_stagger011.cc

```diff
@@ -146,8 +146,8 @@ void restrict_3d_stagger011(
 
   } else {
 
-// Loop over coarse region
-#pragma omp parallel
+    // Loop over coarse region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.